src/lib/ui/CommandPalette.svelte

```
@@ -0,0 +1,397 @@
+										{#each command.shortcut as key}
+											<kbd
+												class="border-base-300 bg-base-200 text-base-content/70 rounded border px-1.5 py-0.5 text-[10px] font-semibold"
+											>
+												{key}
+											</kbd>
+										{/each}
+									</div>
+								{/if}
+							</div>
+						{/each}
+					{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)